lib/src/collections/observable_list.dart

```diff
@@ -25,6 +25,11 @@ import 'package:observable/src/differs.dart';
 ///
 /// *See [ListDiffer] to manually diff two lists instead*
 abstract class ObservableList<E> implements List<E>, Observable {
+  /// An empty observable list that never has changes.
+  static const ObservableList EMPTY = const _ObservableUnmodifiableList(
```

Comment:
I'm not seeing any existing code in this package using shout case. IMO since we don't have to follow precedent it would be nicer to use the 'prefer' style for constants

https://www.dartlang.org/guides/language/effective-dart/style#prefer-using-lowercamelcase-for-constant-names